tools/query_breakdown/src/main/java/com/google/bigquery/InputReader.java

```diff
@@ -10,7 +10,6 @@
  * into queries and data-cleaning if needed.
  */
 public class InputReader {
-
```

Comment:
Pls remove this file from the pull request.

Reply:
removed!